docs/access/macos.rst

```diff
@@ -3,16 +3,19 @@
 MacOS
 #####
 
-To start the **Kolibri** server on macOS, double-click the Kolibri app from the **Launchpad**. 
+#. **Kolibri app is not yet sighed, so the first time you start it, do NOT use Launchpad**. Instead use the **Finder** on your Mac to locate Kolibri app in the **Applications** folder.
```

Comment:
```suggestion
#. **The first time you start the Kolibri application, do NOT use Launchpad**. Instead use the **Finder** on your Mac to locate Kolibri in the **Applications** folder. This is because the application is *unsigned* and needs a new permission to run.
```